app/app_FEATURE_TOGGLE_SDK_045.go

```diff
@@ -2,80 +2,3 @@
 // +build FEATURE_TOGGLE_SDK_045
 
 package app
-
```

Comment:
@Jedi2002 we can actually remove this file altogether

Reply:
Allright  thanks, I just wasn't sure if it breaks something